anchore_engine/clients/grype_wrapper.py

```diff
@@ -492,7 +503,7 @@ def _init_latest_grype_db_engine(
             latest_grype_db_dir, grype_db_version, self.VULNERABILITY_FILE_NAME
         )
         db_connect = self.SQL_LITE_URL_TEMPLATE.format(latest_grype_db_file)
-        latest_grype_db_engine = sqlalchemy.create_engine(db_connect, echo=True)
+        latest_grype_db_engine = sqlalchemy.create_engine(db_connect, echo=False)
```

Comment:
This is not core to the intent of this PR, but was a separate ask to reduce logging from the sql alchemy instance created to query grype db.